src/api/GraphQuery/queryBuilder.ts

```diff
@@ -211,12 +211,12 @@ export interface BaseStrictPropertyFilter<T> extends BasePropertyFilter {
 
 export interface StringStrictPropertyFilter extends BaseStrictPropertyFilter<string[]> {
   propertyType: 'string';
-  operator: '=' | '!=';
+  operator: '=' | '!=' | 'isNull' | 'isNotNull';
 }
 
 export interface NumberStrictPropertyFilter extends BaseStrictPropertyFilter<number> {
   propertyType: 'number';
-  operator: '>' | '<' | '=' | '<=' | '>=' | '!=';
+  operator: '>' | '<' | '=' | '<=' | '>=' | '!=' | 'isNull' | 'isNotNull';
```

Comment:
** Optional suggestion**

We could consider introducing a type for these operators, I think it would be useful in the backend code.

```typescript
// This is just a simplified example, we have actually number and string operators, and some of them are common
export const QUERY_BUILDER__OPERATORS = ['>' | '<' | '=' | '<=' | '>=' | '!=' | 'isNull' | 'isNotNull'] as const;
export type QueryBuilerOperator = (typeof OPERATORS)[number];
```